README.md

```diff
@@ -91,7 +191,34 @@ helm upgrade --install agent-stack-k8s oci://ghcr.io/buildkite/helm/agent-stack-
     --set graphqlToken=<your Buildkite GraphQL-enabled API token>
 ```
 
+Example of the expected output:
+<details>
+
+```
+Release "agent-stack-k8s" does not exist. Installing it now.
+Pulled: ghcr.io/buildkite/helm/agent-stack-k8s:0.20.2
+Digest: sha256:3d3153bfaa2ecbe42db36b6ee241b7f5c0e2f5b04517afb542e5a3d65a6ca597
+NAME: agent-stack-k8s
+LAST DEPLOYED: Thu Dec 19 09:54:47 2024
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+</details>
+
+If you are using [Buildkite Clusters](https://buildkite.com/docs/agent/clusters) to isolate sets of pipelines from each other, you will need to specify the cluster's UUID in the configuration for the controller. This can be done using a flag on the helm command like so:
+
+```
+--set config.cluster-uuid=<your cluster's UUID>
```

Comment:
Since new users will be clusters-enabled, maybe this should be flipped around: `--set config.cluster-uuid` should be in the first example Helm command, with a note below that it is not needed in an unclustered setup.